go.mod

```diff
@@ -66,6 +66,7 @@ require (
 )
 
 require (
+	github.com/golang-jwt/jwt/v5 v5.2.0
```

Comment:
we have v5.0.0 of this package in monorepo. Since it doesn't have other dependencies it maybe OK but please confirm @Groxx.

Reply:
D12725271

Reply:
I think it'll be ok